helm/blueapi/values.yaml

```diff
@@ -75,13 +75,41 @@ worker:
   env:
     sources:
       - kind: deviceFunctions
-        module: blueapi.startup.example
+        module: blueapi.startup.example_devices
+      - kind: planFunctions
+        module: blueapi.startup.example_plans
       - kind: planFunctions
         module: blueapi.plans
   stomp:
     auth:
-      username: guest
-      passcode: guest
-    host: rabbitmq
+      username: foo
+      passcode: bar
```

Comment:
I'm going to make a ticket for describing how to make SealedSecret for BlueAPI and RabbitMQ shared information, I think for this default/example config, we can keep guest/guest and add to the readme for deploying that these values will need to be overwritten if rabbitmq.enabled and not exposed on localhost